src/components/comment/replyInput/ReplyInput.tsx

```diff
@@ -1,36 +1,93 @@
-import React from 'react';
-import { Icon } from '../..';
-import { CommentInputFunctions } from '../../../types';
+import React, { useState } from 'react';
+import { ProfileImage } from '../..';
 import S from './ReplyInput.styled';
+import { Reply } from '../../../assets';
+import { useComment } from '../../../hooks';
+import { useQueryClient } from '@tanstack/react-query';
+import { realpath } from 'fs/promises';
+
+interface ReplyHandler {
+	mention?: string;
+	pageNum: number;
+	groupNumber: number;
+	onClickCancel: () => void;
+	replyInputHandler: () => void;
+}
 
 const ReplyInput = ({
-	contents,
-	addComment,
-	onKeyPress,
-	onChangeHandler,
-	onClickInput,
-}: CommentInputFunctions) => {
-	const isLogin = true; // 임시 코드
+	mention,
+	pageNum,
+	groupNumber,
+	onClickCancel,
+	replyInputHandler,
+}: ReplyHandler) => {
+	const postComment = useComment();
+	const queryClient = useQueryClient();
+	const [contents, setContents] = useState<string>(mention ? `@${mention + ' '}` : '');
+	const userState = JSON.parse(localStorage.getItem('userState') as any);
```

Comment:
```suggestion
	const userInfo = useRecoilValue(userState);
```

로컬스토리지 직접 접근보다는 recoil 이용해서 가져오는게 좋을 것 같습니다!☺️

Reply:
수정했습니다~